src/config.ts

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
+
+export const mySQLConfig = {
```

Comment:
```suggestion
export const mysqlConfig = {
```

we've used this naming style